
---
goal_1: [-0.10, -0.44, 0.16, 0.08, 0.71, -0.0, 0.69]
goal_2: [0.74, -0.01, -0.25, 0.08, 0.70, -0.12, 0.70]
goal_3: [0.45, 0.44, 0.19, 0.02, -0.02, 0.86, 0.51]
goal_4: [0.78, -0.72, 0.35, 0.07, -0.08, -0.63, 0.77]
num_goals: 4